lib/gcp/runtimeconfig.js

```diff
@@ -8,10 +8,28 @@ var _ = require('lodash');
 
 var API_VERSION = 'v1beta1';
 
+function _retryOnServerError(requestFunction) {
+  var retryableCodes = [500, 503];
+  var waitTime = 1000;
+  var wait = function(t) {
```

Comment:
I'm not sure we get anything out of abstracting a promise-ified wait into a helper function. Could just put this directly in the `catch()` block below.